bin/sl-pm.js

```diff
@@ -169,3 +202,12 @@ function stopWhenDone(/* $0, app */) {
   });
   */
 }
+
+function checkAndUpgradeDb(baseDir, callback) {
```

Comment:
Oh, I see now.. so _this_ is the one that should be called `maybeUpgradeDb` since it's either an upgrade or a no-op.